Client/Phpredis/ClientCacheWarmer.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the SncRedisBundle package.
+ *
+ * (c) Henrik Westphal <henrik.westphal@gmail.com>
+ * (c) Yassine Khial <yassine.khial@blablacar.com>
+ * (c) Pierre Boudelle <pierre.boudelle@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Snc\RedisBundle\Client\Phpredis;
+
+use Symfony\Component\HttpKernel\CacheWarmer\CacheWarmerInterface;
+
+/**
+ * @author Javier Spagnoletti <phansys@gmail.com>
+ */
+class ClientCacheWarmer implements CacheWarmerInterface
+{
+    /**
+     * @var ClientBuilder
+     */
+    private $clientBuilder;
+
+    public function __construct(ClientBuilder $clientBuilder)
+    {
+        $this->clientBuilder = $clientBuilder;
+    }
+    public function isOptional()
+    {
+        return class_exists(Client::class);
```

Comment:
Client::class is not imported. Is it normal?

FYI the warmup is not needed if no phpredis clients are used.

Reply:
Testing for `\Redis::class` existence would be enough if the `boot()`  method of the bundle throw an exception like I proposed.